MAUI/DataForm/Overview.md

```diff
@@ -0,0 +1,19 @@
+ **SfDataForm**
+
+The Syncfusion .NET MAUI DataForm (SfDataForm) control is used to create or edit data form such as data form such as login, reservation, contact, employee form and so, based on the business purpose. 
+
+## Key features includes the following:
+
+*  **Layout and grouping**: Supports to linear and grid layout with grouping support. Supports customizing the layout with different heights for each item.
```

Comment:
* **Editors**: Support  built-in editors such as text, password, multi-line, combo box, autocomplete, date, time, checkbox, switch, and radio group editors the primitive data types such as string, enumeration, DateTime, and TimeSpan.

* **Custom editor**: Support to add custom editors.

* **Validation**: Built-in support to validate the data or user input to alert the user to enter proper value. 

* **Commit mode**:   Support to determine when the value should be committed to the  underlying data object.

*  **Layout and grouping**: Supports grouping, linear, grid layout.

* **Appearance customization **: Supports to customize the appearances of editors, labels and group headers.